hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/diskUsage/diskUsage.tsx

```diff
@@ -295,165 +295,167 @@ export class DiskUsage extends React.Component<Record<string, object>, IDUState>
       keys.push('Entity Type');
       values.push(summaryResponse.type);
 
-      if (summaryResponse.countStats.type === 'KEY') {
-        const keyEndpoint = `/api/v1/namespace/du?path=${path}&replica=true`;
-        const { request: metadataRequest, controller: metadataNewController } = AxiosGetHelper(keyEndpoint, cancelKeyMetadataSignal);
-        cancelKeyMetadataSignal = metadataNewController;
-        metadataRequest.then(response => {
-          keys.push('File Size');
-          values.push(byteToSize(response.data.size, 3));
-          keys.push('File Size With Replication');
-          values.push(byteToSize(response.data.sizeWithReplica, 3));
-          console.log(values);
-
-          this.setState({
-            showPanel: true,
-            panelKeys: keys,
-            panelValues: values
+      if (summaryResponse.status !== 'INITIALIZING') {
+        if (summaryResponse.countStats.type === 'KEY') {
+          const keyEndpoint = `/api/v1/namespace/du?path=${path}&replica=true`;
```

Comment:
Add comments to explain the purpose of the checks for `INITIALIZING` status. This will help future maintainers understand the rationale behind these conditions.

Reply:
@ArafatKhan2198 
Done changes in latest commit 8b268709451b647d34280440097d220fff5ddada